Mage.Client/src/main/java/mage/client/dialog/NewTournamentDialog.java

```diff
@@ -1202,6 +1228,18 @@ private TournamentOptions getTournamentOptions() {
             tOptions.getLimitedOptions().setIsRandom(tournamentType.isRandom());
             tOptions.getLimitedOptions().setIsRichMan(tournamentType.isRichMan());
             tOptions.getLimitedOptions().setIsJumpstart(tournamentType.isJumpstart());
+
+            if (tournamentType.isJumpstart()) {
+                if (!(jumpstartPacksFilename.isEmpty())) {
+                    String jumpstartPacksData = "";
+                    try {
+                        jumpstartPacksData = new String(Files.readAllBytes(Paths.get(jumpstartPacksFilename)));
```

Comment:
Must have data size check for uploaded data (example: if string length > 500000 then show error on too big file)

Reply:
Modifed as requested